### DISCUSSION TITLE
Skeleton Labs is now available on Bluesky! 🦋

### DISCUSSION THREAD

**Discussion by endigo9740** [skeletonlabs/skeleton#2792] [Announcements]
![banner](https://github.com/user-attachments/assets/2fbf9aa3-60b3-4606-a72d-07cead419704)

Hey everyone, Chris here from Skeleton Labs. Today I'm very excited to share that Skeleton Labs is now available on Bluesky.

🦋 https://bsky.app/profile/skeleton.dev

You'll now see a link to this service in the footer of our doc sites (the Butterfly icon):

![Screenshot 2024-08-02 at 3 10 26 PM](https://github.com/user-attachments/assets/0aa43fd7-12d8-4dc5-85d0-fad6f8760f40)

## What is Bluesky?

If you're not familiar with [Bluesky](https://bsky.social/about), it's one of several alternatives to X (Twitter) that has appeared over the last few years. Similar to [Threads](https://www.threads.net/) or [Mastodon](https://joinmastodon.org/), it provides a [federated social network](https://en.wikipedia.org/wiki/Fediverse) focused on microblogging. This also comes with a number of really cool features out of the box:

- It's built around the federated [At Protocol](https://atproto.com/)
- Their web client is open source and [available on GitHub](https://github.com/bluesky-social/social-app)
- They provide open APIs and developer tools: https://docs.bsky.app/
- The web and mobile clients feature a simple and easy to use interface that feels very familiar
- All accounts have the option of being set public. See more on this below.

## No Account Needed

Don't worry if you prefer to stick with another social network - Bluesky does not require you to have an account. This means you get full access to both our profile and posts with no login required. Seriously, [try it out](https://bsky.app/profile/skeleton.dev)!

## How will Skeleton Labs use Bluesky?

![bsky-screenshot](https://github.com/user-attachments/assets/0e4074e4-0859-43cc-bf96-a60c4f417a8b)

For the most part we plan to use it as a drop in replace for X/Twitter. Providing regular updates for releases and announcements week to week. Additionally, you may see also see a new Bluesky bot channel appear on Discord. And we hope to make use of their open APIs to populate News sections on both the [Skeleton v3](https://next.skeleton.dev/) and [Floating UI Svelte](https://floating-ui-svelte.vercel.app/) doc sites in the near future.

## Why leave X/Twitter?

While I know some folks will have strong opinions on this subject, we no longer feel that X (aka Twitter) adheres to the standards the team and I hold ourselves to - many of which we cover in our [Code of Conduct](https://github.com/skeletonlabs/community/blob/main/CODE_OF_CONDUCT.md) (based on those by [Svelte](https://github.com/sveltejs/community/blob/main/CODE_OF_CONDUCT.md)). While no social network is inherently good or bad, we've been monitoring the changes over the last year  as Twitter rebranded to X. Unfortunately we just no longer feel their platform is the right fit for us.

Not to mention we're an open source team. We build open source software. So the decision to support open platforms with open source tooling is right at the forefront for us. We may even explore additional options like Mastodon in the future for the same reasons.

## Please keep your feedback respectful

That said, we would prefer this thread not become a forum for discussing the follies of social networks. If you have an opinion that differs from our own, or you think we're making a mistake, we can respect that. If you feel strongly enough about this, please feel free to reach out to me (Chris) either via a direct message in the Skeleton Discord (@endigo9740) or via my email at chris@skeleton.dev. I promise to read and respond to any and all messages that are respectful and provided in good faith.

**Comment by gzuuus** [comment]
Have you heard of Nostr? It offers a better alternative to BSK when it comes to social media that you control, and also presents endless possibilities for developers to integrate with it.

You can learn more at [nostr.com](http://nostr.com/) and [nostr.how](http://nostr.how/).

Additionally, I'd like to share a couple of articles that might be interesting in exploring this further. [1](https://habla.news/a/naddr1qvzqqqr4gupzpu9v5anc855x7htnx6cj3zy0thel9x0qwxtnatcru5sqj8wwxvrwqqk9w6re94pxcat9wd4hjt2hd9kxct2swfhkyctzd3uj6nn9wejhyt2zv5k5japdxpnn2ut8wquxpwtd) [2](https://fiatjaf.com/ab1127fb.html)

**Reply by endigo9740** [reply]
I have heard of it, but haven't looked into in detail. Thanks for sharing the links though. I'll check it out.

We do plan to use other options in the future, we just try to keep the scope slim because we would rather focus on a small handful of networks and ensure we're active on those. Rather than try to use the shotgun method.

Options like Mastodon are highly likely at some point for example.

**Reply by gzuuus** [reply]
Definitely, I understand it - there are so many options and so many messaging apps out there! XD Nostr itself is built on asymmetrical encryption, which means your identity is tied to a unique key pair that you can easily generate offline. This ensures that the identity you create belongs solely to you, and no one else can use it to sign notes in your name - a very powerful feature, especially given the growing need to prevent impersonation.

Nostr stands out as the first open social media protocol, unlike other platforms which are often 'walled gardens' prone to abuse, censorship, or 'enshittification.'
